src/main/resources/application.properties

```diff
@@ -1,14 +1,18 @@
 spring.application.name=stars
+spring.devtools.restart.enabled=true
 
 # PostgreSQL database configuration
-spring.datasource.url=jdbc:postgresql://localhost:5432/f2p-starhunt?currentSchema=f2p_starhunt
+spring.datasource.url=jdbc:postgresql://postgresql-20240401225708.postgres.database.azure.com:5432/postgres?sslmode=require
```

Comment:
This is the production DB I suppose? Can we have a development profile which points to localhost, and keep this application.properties file for production?

Reply:
Alterntavely we can also template the database url using an environment variable I guess :)

Reply:
> This is the production DB I suppose? Can we have a development profile which points to localhost, and keep this application.properties file for production?

Hey @Jannyboy11 just added this